DOCKER.md

```diff
@@ -53,15 +54,18 @@ For example, "What is a maker schedule?"
 
 ### 4. Run the application
 
-Execute the run script from the root directory, specifying your configuration as `config/pg-search`, and selecting the `default` profile from your secrets file:
+Execute the run script from the root Vectara Answer folder, specifying your configuration as `config/pg-search`, and selecting the `default` profile from your secrets file:
```

Comment:
Nit: I think either "project's root folder" or "Vectara Answer root folder" would be a bit clearer.